### DISCUSSION TITLE
How can I type a forwardslash circle symbol?

### DISCUSSION THREAD

**Discussion by ForBo7** [typst/typst#1642] [Q&A] [answered]
So a div circle symbol exists:

<img width="99" alt="image" src="https://github.com/typst/typst/assets/85829792/5175b736-368f-4ef0-bc5f-662e902dc889">

And so does a backwardslash circle symbol:

<img width="98" alt="image" src="https://github.com/typst/typst/assets/85829792/30f20836-a943-4ce2-9b60-d20e44138e20">

Does a forwardslash symbol exist? If not, is there a way to type one?

**Comment by Enivex** [comment, marked as answer]
There doesn't seem to be a name for it yet, but the symbol can be accessed with `\u{2298}`
